eng/pipelines/templates/steps/build-conda-artifacts.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
+  - name: TestPipeline
+    type: boolean
+    default: false
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: CondaArtifacts
+    type: object
+    default: []
+
+steps:
+  - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+    parameters:
+      PackageName: "azure-template"
+      ServiceDirectory: "template"
+      TestPipeline: ${{ parameters.TestPipeline }}
```

Comment:
Should `PackageName` and `ServiceDirectory` be passed in parameters?

Reply:
Honestly, I think I can just rip this entire thing out.

Reply:
I leave it to you Scott but it might still be interesting to have the template for testing things here.